### PR TITLE
Translate Code-Splitting

### PR DESCRIPTION
**Pull request by masbagal**
Thanks for the opportunity.

BTW, I cannot find the nice translation for `code-splitting` so I leave it as it is. I've tried `pemecahan kode` and `pembagian kode` but it didn't felt right. Finally left it as `code-splitting` also thinking that should someone wants to dive deeper, using the term `code-splitting` as a search keyword will make the search easier. 

Please comment if you have suggestion :bowing_man:

### CODE REVIEW

File: content/docs/code-splitting.md [hunk 1]
```diff
@@ -247,21 +213,21 @@ const App = () => (
...
 
-`React.lazy` currently only supports default exports. If the module you want to import uses named exports, you can create an intermediate module that reexports it as the default. This ensures that tree shaking keeps working and that you don't pull in unused components.
+`React.lazy` sementara ini hanya dapat digunakan untuk modul dengan ekspor `default`. Jika modul yang ingin Anda impor menggunakan *named exports* (ekspor modul yang diberi nama dan tidak menggunakan *default*), Anda dapat membuat modul perantara yang mengekspor ulang modul tersebut sebagai *default*. Cara ini memastikan proses *treeshaking* tetap berjalan dan tidak mengikutsertakan komponen yang tidak terpakai.
 
 ```js
-// ManyComponents.js
+// KumpulanKomponen.js
 export const MyComponent = /* ... */;
-export const MyUnusedComponent = /* ... */;
+export const KomponenTidakTerpakai = /* ... */;
```
Comment (resir014) [inline content/docs/code-splitting.md]:
```suggestion
export const MyUnusedComponent = /* ... */;
```

Please don't translate file + variable names, as mentioned in the [translation guidelines](https://github.com/reactjs/id.reactjs.org/blob/master/TRANSLATION.md).

File: content/docs/code-splitting.md [hunk 2]
```diff
@@ -247,21 +213,21 @@ const App = () => (
 
 ## Named Exports {#named-exports}
 
-`React.lazy` currently only supports default exports. If the module you want to import uses named exports, you can create an intermediate module that reexports it as the default. This ensures that tree shaking keeps working and that you don't pull in unused components.
+`React.lazy` sementara ini hanya dapat digunakan untuk modul dengan ekspor `default`. Jika modul yang ingin Anda impor menggunakan *named exports* (ekspor modul yang diberi nama dan tidak menggunakan *default*), Anda dapat membuat modul perantara yang mengekspor ulang modul tersebut sebagai *default*. Cara ini memastikan proses *treeshaking* tetap berjalan dan tidak mengikutsertakan komponen yang tidak terpakai.
 
 ```js
-// ManyComponents.js
+// KumpulanKomponen.js
```
Comment (resir014) [inline content/docs/code-splitting.md]:
```suggestion
// ManyComponents.js
```

idem.

File: content/docs/code-splitting.md [hunk 3]
```diff
@@ -247,21 +213,21 @@ const App = () => (
...
 export const MyComponent = /* ... */;
-export const MyUnusedComponent = /* ... */;
+export const KomponenTidakTerpakai = /* ... */;
 ```
 
 ```js
 // MyComponent.js
-export { MyComponent as default } from "./ManyComponents.js";
+export { MyComponent as default } from "./KumpulanKomponen.js";
```
Comment (resir014) [inline content/docs/code-splitting.md]:
```suggestion
export { MyComponent as default } from "./ManyComponents.js";
```

Same regarding file + variable names.

File: content/docs/code-splitting.md [hunk 4]
```diff
@@ -247,21 +213,21 @@ const App = () => (
...
 ```
 
 ```js
-// MyApp.js
+// AplikasiSaya.js
```
Comment (resir014) [inline content/docs/code-splitting.md]:
```suggestion
// MyApp.js
```

idem.

File: content/docs/code-splitting.md [hunk 5]
```diff
@@ -4,122 +4,95 @@ title: Code-Splitting
 permalink: docs/code-splitting.html
 ---
 
-## Bundling {#bundling}
+## Bundel {#bundling}
+Kebanyakan aplikasi React akan "membundel" *file* menggunakan alat bantu seperti [Webpack](https://webpack.js.org/) atau [Browserify](http://browserify.org/). Pembundelan ini adalah sebuah proses yang menelusuri sejumlah *file* yang terimpor dan digabungkan menjadi sebuah file: sebuah "bundel". Bundel ini kemudian dapat digunakan dalam halaman web untuk memuat keseluruhan aplikasi sekaligus.
```
Comment (resir014) [inline content/docs/code-splitting.md]:
```suggestion

Kebanyakan aplikasi React akan "membundel" *file* menggunakan alat bantu seperti [Webpack](https://webpack.js.org/) atau [Browserify](http://browserify.org/). Pembundelan ini adalah sebuah proses yang menelusuri sejumlah *file* yang terimpor dan digabungkan menjadi sebuah file: sebuah "bundel". Bundel ini kemudian dapat digunakan dalam halaman web untuk memuat keseluruhan aplikasi sekaligus.
```

Nit: removing the extra line here will mess with merging when we sync from the root repo.

**Comment by masbagal** [comment]
Terimakasih untuk reviewnya. 
Updated, minta tolong bantuannya untuk dicek ulang @resir014

**Pull request merged**